tests/Sdk/Integration/SpanProcessorTest.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OpenTelemetry\Tests\Sdk\Integration;
+
+use OpenTelemetry\Context\Context;
+use OpenTelemetry\Sdk\Trace\SpanProcessor;
+use OpenTelemetry\Sdk\Trace\TracerProvider;
+use OpenTelemetry\Trace\Span;
+use PHPUnit\Framework\TestCase;
+
+class SpanProcessorTest extends TestCase
+{
+    /**
+     * @test
+     */
+    public function parentContextShouldBePassedToSpanProcessor()
+    {
+        $parentContext = new Context();
```

Comment:
This looks great, thanks for putting this PR up!  Would it make sense to have a test case where parentContext is null and/or not passed in?

Reply:
Yes, I've added a test case when `null` is passed as a parentContext argument on Span creation. The current Context is used in this case and passed to the SpanProcessor.